### PR TITLE
created models

### PR DESCRIPTION
**Pull request by bilol0711**
plese team check models

### CODE REVIEW

**Review by EnU098 — APPROVED** [review]

**Review by Baxodir0327 — APPROVED** [review]

File: src/main/java/com/company/model/Backed.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.company.model;
+
+public class Backed {
```
Comment (axrortatu) [inline src/main/java/com/company/model/Backed.java:3]: what does it mean Backed ?

**Review by EnU098 — APPROVED** [review]

**Review by Baxodir0327 — APPROVED** [review]

File: src/main/java/com/company/server/enums/State.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.company.server.enums;
+//Akmal aka
+public enum State {
+    PHONE_NUMBER, MAIN_PAGE, CHOOSE_CATEGORY, ENTER_CATEGORY_NAME, ENTER_PRODUCT_NAME, CHOOSE_PRODUCT, ENTER_PRODUCT_PRICE, ENTER_PRODUCT_URL, ENTER_NAME
+
```
Comment (axrortatu) [inline src/main/java/com/company/server/enums/State.java:5]: remove unnecessary line

File: src/main/java/com/company/server/model/User.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import com.company.server.enums.State;
+import lombok.*;
+
+    @EqualsAndHashCode(callSuper = true)
+    @Data
+    @NoArgsConstructor
+    @AllArgsConstructor
+    @Builder
+    public class User extends BaseModel{
```
Comment (axrortatu) [inline src/main/java/com/company/server/model/User.java:11]: why we need tabs here

**Review by Baxodir0327 — APPROVED** [review]
good

**Review by EnU098 — APPROVED** [review]
ok

File: src/main/java/com/company/server/model/User.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class User extends BaseModel{
+        private String fullName;
+        private String username;
+        private String phoneNumber;
+        private Long chatId;
+        private State state;
+    }
```
Comment (axrortatu) [inline src/main/java/com/company/server/model/User.java:18]: why we need spaces

**Review by axrortatu — APPROVED** [review]

**Review by Akmal-85 — APPROVED** [review]

**Pull request merged**